tests/live/test_network.py

```diff
@@ -379,11 +379,10 @@ def update_state_obj(self, fw, state):
         state.obj.comment = 'This is an update layer3 interface'
 
     def cleanup_dependencies(self, fw, state):
-        for x in state.management_profiles:
-            try:
-                x.delete()
-            except Exception:
-                pass
+        try:
+            state.management_profiles[0].delete_similar()
+        except Exception:
```

Comment:
It's usually bad practice to have `except Exception:`  Can you be more specific about which exception to pass?